src/config/dark-sites.config

```diff
@@ -503,6 +503,7 @@ marvil.co
 mastercomfig.com
 mastodon.online
 mastodon.social
+https://messages.google.com/web
```

Comment:
Is it dark by default? If not, it shouldn't be in this list. Also, please don't use https:// and preserve the alphabetical order.